packages/module/tsconfig.json

```diff
@@ -25,9 +25,9 @@
     // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
 
     /* Modules */
-    "module": "ESNext",                                  /* Specify what module code is generated. */
+    "module": "Node16",                                  /* Specify what module code is generated. */
```

Comment:
This had to be changed as a consequence of changing `moduleResolution` to `Node16`.